src-docs/src/views/xy_chart_axis/simple_axis.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+import {
+  EuiLineSeries,
+  EuiXAxis,
+  EuiYAxis,
+  EuiXYChart,
+  EuiXYChartAxisUtils,
+  EuiXYChartTextUtils,
+} from '../../../../src/components';
+
+const DATA = [{ x: 0, y: 5 }, { x: 1, y: 3 }, { x: 2, y: 2 }, { x: 3, y: 3 }];
+
+function xAxisTickFormatter(value) {
+  return EuiXYChartTextUtils.tspanTextWrapper(['MY VALUE', value]);
+}
+
+export default () => (
+  <EuiXYChart width={600} height={200} xPadding={10} yPadding={10} showDefaultAxis={false}>
+    <EuiLineSeries name="Total Bytes" data={DATA} />
+    <EuiYAxis title="Left Y Axis" tickLabelAngle={45} />
+    <EuiYAxis
+      title="Right Y Axis"
+      orientation={EuiXYChartAxisUtils.ORIENTATION.RIGHT}
+      tickLabelAngle={-45}
+    />
+    <EuiXAxis
+      title="Bottom X Axis"
+      titlePosition={EuiXYChartAxisUtils.TITLE_POSITION.START}
+      tickFormat={xAxisTickFormatter}
+    />
+    <EuiXAxis
+      title="Top X Axis"
```

Comment:
top axis text is flipped upside down

Reply:
Do you mean the tick values? if so it's only uses to show all the possible rotation (I know 180 degree rotation doesn't make sense, but it was just for the example. I will change to show a simple 45 degree rotation

Reply:
Yeah, the tick values. 45 or 90 degree angle would be nice, I think it's good to keep the examples showing off the pretty uses of each component :)